util/model_output_parser.py

```diff
@@ -60,6 +61,80 @@ def parse_netcdf(file_id, filename, specification):
     ds.close()
     return var_dfs
 
+def parse_grib(file_id, filename, specification):
+    logger.debug("parse GRIB file {}".format(filename))
+    time_dim = specification["time_dim"]
+    horizontal_dims = specification["horizontal_dims"]
+    fill_value_key = specification.get("fill_value_key", None)
```

Comment:
This seems to be never used, do we actually need it?

Reply:
It is used as an argument in the function "dataframe_from_ncfile".

Reply:
Ah yes, I missed that.